### ISSUE TITLE
Emulator crashes / reboots when opening settings app

### ISSUE THREAD

**Issue by Kignuf** [butomo1989/docker-android#74]
Operating System: 
Ubuntu 16.04

Docker Image:
butomo1989/docker-android-x86-7.1.1

Docker Version:  
Docker version 18.03.0-ce, build 0520e24

Docker-compose version (Only if you use it):  
docker-compose version 1.20.1, build 5d8c71b

Docker Command to start docker-android:  
I'm running following docker-compose file
```
version: "2.2"

services:
  selenium_hub:
    image: selenium/hub:3.11.0-bismuth
    ports:
      - "4444:4444"
  
  nexus_7.1.1:
    image: butomo1989/docker-android-x86-7.1.1
    privileged: true
    scale: 3
    depends_on:
      - selenium_hub
    ports:
      - 6080
    volumes:
      - $PWD/happn-rc-latest.apk:/root/tmp
    environment:
      - DEVICE=Nexus 5
      - CONNECT_TO_GRID=True
      - APPIUM=True
      - SELENIUM_HOST=selenium_hub
      - AUTO_RECORD=True
```

## Steps to reproduce
* Open browser on http://localhost:6080
* In the emulator, run the settings app

## Expected Behavior
Settings app opens
Android emulator does not crash / reboot

## Actual Behavior
Android emulator crashes and restarts.
See logs below

```
Cmd line: com.google.android.gms.persistent
Build fingerprint: 'Android/sdk_google_phone_x86_64/generic_x86_64:7.1.1/NYC/4498512:userdebug/test-keys'
ABI: 'x86_64'
Build type: optimized
Zygote loaded classes=4375 post zygote classes=429
Intern table: 42044 strong; 217 weak
JNI: CheckJNI is on; globals=513 (plus 113 weak)
Libraries: /data/user_de/0/com.google.android.gms/app_extracted_libs/x86_64/libconscrypt_gmscore_jni.so /data/user_de/0/com.google.android.gms/app_extracted_libs/x86_64/libgmscore.so /system/lib64/libandroid.so /system/lib64/libcompiler_rt.so /system/lib64/libjavacrypto.so /system/lib64/libjnigraphics.so /system/lib64/libmedia_jni.so /system/lib64/libwebviewchromium_loader.so libjavacore.so libopenjdk.so (10)
Heap: 37% free, 6MB/9MB; 32325 objects
Dumping cumulative Gc timings
Start Dumping histograms for 1 iterations for sticky concurrent mark sweep
MarkRootsCheckpoint:	Sum: 250ms 99% C.I. 80ms-170ms Avg: 125ms Max: 170ms
ScanGrayAllocSpaceObjects:	Sum: 90ms 99% C.I. 0.010ms-89.408ms Avg: 22.500ms Max: 90ms
FreeList:	Sum: 20ms 99% C.I. 2.034us-14499.999us Avg: 338.983us Max: 20000us
(Paused)PausePhase:	Sum: 0 99% C.I. 0ns-0ns Avg: 0ns Max: 0ns
Done Dumping histograms
sticky concurrent mark sweep paused:	Sum: 0 99% C.I. 0ns-0ns Avg: 0ns Max: 0ns
sticky concurrent mark sweep total time: 360ms mean time: 360ms
sticky concurrent mark sweep freed: 58711 objects with total size 4MB
sticky concurrent mark sweep throughput: 163086/s / 13MB/s
Total time spent in GC: 360ms
Mean GC size throughput: 12MB/s
Mean GC object throughput: 163064 objects/s
Total number of allocations 91028
Total bytes allocated 10MB
Total bytes freed 4MB
Free memory 3MB
Free memory until GC 3MB
Free memory until OOME 249MB
Total memory 9MB
Max memory 256MB
Zygote space size 1088KB
Total mutator paused time: 0
Total time waiting for GC to complete: 0
Total GC count: 1
Total GC time: 360ms
Total blocking GC count: 0
Total blocking GC time: 0
Histogram of GC count per 10000 ms: 0:2
Histogram of blocking GC count per 10000 ms: 0:2
Histogram of native allocation 0:599,128:252,256:114,384:67,640:1,896:16,1152:2 bucket size 128
Histogram of native free 0:7,64:59,192:227,256:28,384:10,448:12,768:1 bucket size 64
/system/framework/oat/x86_64/com.android.location.provider.odex: speed
/system/framework/oat/x86_64/com.android.media.remotedisplay.odex: speed
/system/priv-app/PrebuiltGmsCore/oat/x86_64/PrebuiltGmsCore.odex: speed
Current JIT code cache size: 0B
Current JIT data cache size: 5KB
Current JIT capacity: 64KB
Current number of JIT code cache entries: 0
Total number of JIT compilations: 0
Total number of JIT compilations for on stack replacement: 0
Total number of deoptimizations: 0
Total number of JIT code cache collections: 0
Memory used for stack maps: Avg: --9223372036854775808B Max: 0B Min: -1B
Memory used for compiled code: Avg: --9223372036854775808B Max: 0B Min: -1B
Memory used for profiling info: Avg: 5KB Max: 5KB Min: 5KB
Start Dumping histograms for 1 iterations for JIT timings
Compiling:	Sum: 0 99% C.I. 0ns-0ns Avg: 0ns Max: 0ns
Done Dumping histograms
Memory used for compilation: Avg: --9223372036854775808B Max: 0B Min: -1B

suspend all histogram:	Sum: 10ms 99% C.I. 1.365us-9441.279us Avg: 625us Max: 10000us
DALVIK THREADS (13):
"Signal Catcher" daemon prio=5 tid=3 Runnable
  | group="system" sCount=0 dsCount=0 obj=0x12c31670 self=0x76205f1c8400
  | sysTid=29875 nice=0 cgrp=default sched=0/0 handle=0x76206790d450
  | state=R schedstat=( 0 0 0 ) utm=2 stm=0 core=0 HZ=100
  | stack=0x762067811000-0x762067813000 stackSize=1013KB
  | held mutexes= "mutator lock"(shared held)
  native: #00 pc 0000000000531a6e  /system/lib64/libart.so (_ZN3art15DumpNativeStackERNSt3__113basic_ostreamIcNS0_11char_traitsIcEEEEiP12BacktraceMapPKcPNS_9ArtMethodEPv+238)
  native: #01 pc 000000000050278c  /system/lib64/libart.so (_ZNK3art6Thread9DumpStackERNSt3__113basic_ostreamIcNS1_11char_traitsIcEEEEbP12BacktraceMap+412)
  native: #02 pc 000000000051c418  /system/lib64/libart.so (_ZN3art14DumpCheckpoint3RunEPNS_6ThreadE+936)
  native: #03 pc 000000000051381f  /system/lib64/libart.so (_ZN3art10ThreadList13RunCheckpointEPNS_7ClosureE+511)
  native: #04 pc 0000000000513420  /system/lib64/libart.so (_ZN3art10ThreadList4DumpERNSt3__113basic_ostreamIcNS1_11char_traitsIcEEEEb+848)
  native: #05 pc 000000000051308a  /system/lib64/libart.so (_ZN3art10ThreadList14DumpForSigQuitERNSt3__113basic_ostreamIcNS1_11char_traitsIcEEEE+826)
  native: #06 pc 00000000004ec973  /system/lib64/libart.so (_ZN3art7Runtime14DumpForSigQuitERNSt3__113basic_ostreamIcNS1_11char_traitsIcEEEE+195)
  native: #07 pc 00000000004f3d97  /system/lib64/libart.so (_ZN3art13SignalCatcher13HandleSigQuitEv+2375)
  native: #08 pc 00000000004f2828  /system/lib64/libart.so (_ZN3art13SignalCatcher3RunEPv+488)
  native: #09 pc 00000000000897f1  /system/lib64/libc.so (_ZL15__pthread_startPv+177)
  native: #10 pc 0000000000029a6b  /system/lib64/libc.so (__start_thread+11)
  native: #11 pc 000000000001cae5  /system/lib64/libc.so (__bionic_clone+53)
  (no managed stack frames)

"Jit thread pool worker thread 0" prio=5 tid=2 Native (still starting up)
  | group="" sCount=1 dsCount=0 obj=0x0 self=0x762068496400
  | sysTid=29874 nice=9 cgrp=default sched=0/0 handle=0x762067a0e450
  | state=S schedstat=( 0 0 0 ) utm=0 stm=0 core=0 HZ=100
  | stack=0x762067910000-0x762067912000 stackSize=1021KB
  | held mutexes=
  kernel: futex_wait_queue_me+0xdb/0x123
  kernel: futex_wait+0x11c/0x261
  kernel: do_futex+0xc7/0x999
  kernel: SyS_futex+0xbd/0x124
  kernel: system_call_fastpath+0x16/0x1b
  native: #00 pc 000000000001ccc6  /system/lib64/libc.so (syscall+22)
  native: #01 pc 000000000014f12b  /system/lib64/libart.so (_ZN3art17ConditionVariable16WaitHoldingLocksEPNS_6ThreadE+123)
  native: #02 pc 000000000051e4fa  /system/lib64/libart.so (_ZN3art10ThreadPool7GetTaskEPNS_6ThreadE+298)
  native: #03 pc 000000000051d861  /system/lib64/libart.so (_ZN3art16ThreadPoolWorker3RunEv+97)
  native: #04 pc 000000000051d1a6  /system/lib64/libart.so (_ZN3art16ThreadPoolWorker8CallbackEPv+86)
  native: #05 pc 00000000000897f1  /system/lib64/libc.so (_ZL15__pthread_startPv+177)
  native: #06 pc 0000000000029a6b  /system/lib64/libc.so (__start_thread+11)
  native: #07 pc 000000000001cae5  /system/lib64/libc.so (__bionic_clone+53)
  (no managed stack frames)

"JDWP" daemon prio=5 tid=4 WaitingInMainDebuggerLoop
  | group="system" sCount=1 dsCount=0 obj=0x12c31700 self=0x76205f192800
  | sysTid=29876 nice=0 cgrp=default sched=0/0 handle=0x76206780e450
  | state=S schedstat=( 0 0 0 ) utm=0 stm=0 core=0 HZ=100
  | stack=0x762067712000-0x762067714000 stackSize=1013KB
  | held mutexes=
  kernel: unix_stream_recvmsg+0x34d/0x83b
  kernel: sock_recvmsg+0xac/0xcd
  kernel: ___sys_recvmsg+0x155/0x21a
  kernel: __sys_recvmsg+0x3d/0x5e
  kernel: SyS_recvmsg+0x14/0x16
  kernel: system_call_fastpath+0x16/0x1b
  native: #00 pc 000000000008d645  /system/lib64/libc.so (recvmsg+5)
  native: #01 pc 0000000000619840  /system/lib64/libart.so (_ZN3art4JDWP12JdwpAdbState15ReceiveClientFdEv+176)
  native: #02 pc 0000000000619b67  /system/lib64/libart.so (_ZN3art4JDWP12JdwpAdbState6AcceptEv+567)
  native: #03 pc 0000000000398120  /system/lib64/libart.so (_ZN3art4JDWP9JdwpState3RunEv+368)
  native: #04 pc 00000000003977b8  /system/lib64/libart.so (_ZN3art4JDWPL15StartJdwpThreadEPv+40)
  native: #05 pc 00000000000897f1  /system/lib64/libc.so (_ZL15__pthread_startPv+177)
  native: #06 pc 0000000000029a6b  /system/lib64/libc.so (__start_thread+11)
  native: #07 pc 000000000001cae5  /system/lib64/libc.so (__bionic_clone+53)
  (no managed stack frames)

"Binder:29869_1" prio=5 tid=5 Native
  | group="main" sCount=1 dsCount=0 obj=0x12c31a60 self=0x762068469e00
  | sysTid=29881 nice=0 cgrp=default sched=0/0 handle=0x76204f96b450
  | state=S schedstat=( 0 0 0 ) utm=0 stm=0 core=0 HZ=100
  | stack=0x76204f86f000-0x76204f871000 stackSize=1013KB
  | held mutexes=
  kernel: binder_thread_read+0xdb8/0xfe0
  kernel: binder_ioctl_write_read+0x16c/0x28b
  kernel: binder_ioctl+0x1ea/0x5bd
  kernel: do_vfs_ioctl+0x37d/0x427
  kernel: SyS_ioctl+0x59/0x7d
  kernel: system_call_fastpath+0x16/0x1b
  native: #00 pc 000000000008c825  /system/lib64/libc.so (__ioctl+5)
  native: #01 pc 000000000002b7a8  /system/lib64/libc.so (ioctl+56)
  native: #02 pc 000000000005f49e  /system/lib64/libbinder.so (_ZN7android14IPCThreadState14talkWithDriverEb+270)
  native: #03 pc 000000000005f5f0  /system/lib64/libbinder.so (_ZN7android14IPCThreadState20getAndExecuteCommandEv+16)
  native: #04 pc 000000000005fd7f  /system/lib64/libbinder.so (_ZN7android14IPCThreadState14joinThreadPoolEb+79)
  native: #05 pc 0000000000080b67  /system/lib64/libbinder.so (_ZN7android10PoolThread10threadLoopEv+23)
  native: #06 pc 0000000000012c99  /system/lib64/libutils.so (_ZN7android6Thread11_threadLoopEPv+313)
  native: #07 pc 00000000000ab0d3  /system/lib64/libandroid_runtime.so (_ZN7android14AndroidRuntime15javaThreadShellEPv+99)
  native: #08 pc 00000000000897f1  /system/lib64/libc.so (_ZL15__pthread_startPv+177)
  native: #09 pc 0000000000029a6b  /system/lib64/libc.so (__start_thread+11)
  native: #10 pc 000000000001cae5  /system/lib64/libc.so (__bionic_clone+53)
  (no managed stack frames)

"Binder:29869_2" prio=5 tid=6 Native
  | group="main" sCount=1 dsCount=0 obj=0x12c31af0 self=0x76205f1b4200
  | sysTid=29882 nice=0 cgrp=default sched=0/0 handle=0x76204f86c450
  | state=S schedstat=( 0 0 0 ) utm=0 stm=1 core=0 HZ=100
  | stack=0x76204f770000-0x76204f772000 stackSize=1013KB
  | held mutexes=
  kernel: binder_thread_read+0xdb8/0xfe0
  kernel: binder_ioctl_write_read+0x16c/0x28b
  kernel: binder_ioctl+0x1ea/0x5bd
  kernel: do_vfs_ioctl+0x37d/0x427
  kernel: SyS_ioctl+0x59/0x7d
  kernel: system_call_fastpath+0x16/0x1b
  native: #00 pc 000000000008c825  /system/lib64/libc.so (__ioctl+5)
  native: #01 pc 000000000002b7a8  /system/lib64/libc.so (ioctl+56)
  native: #02 pc 000000000005f49e  /system/lib64/libbinder.so (_ZN7android14IPCThreadState14talkWithDriverEb+270)
  native: #03 pc 000000000005f5f0  /system/lib64/libbinder.so (_ZN7android14IPCThreadState20getAndExecuteCommandEv+16)
  native: #04 pc 000000000005fd7f  /system/lib64/libbinder.so (_ZN7android14IPCThreadState14joinThreadPoolEb+79)
  native: #05 pc 0000000000080b67  /system/lib64/libbinder.so (_ZN7android10PoolThread10threadLoopEv+23)
  native: #06 pc 0000000000012c99  /system/lib64/libutils.so (_ZN7android6Thread11_threadLoopEPv+313)
  native: #07 pc 00000000000ab0d3  /system/lib64/libandroid_runtime.so (_ZN7android14AndroidRuntime15javaThreadShellEPv+99)
  native: #08 pc 00000000000897f1  /system/lib64/libc.so (_ZL15__pthread_startPv+177)
  native: #09 pc 0000000000029a6b  /system/lib64/libc.so (__start_thread+11)
  native: #10 pc 000000000001cae5  /system/lib64/libc.so (__bionic_clone+53)
  (no managed stack frames)

"HeapTaskDaemon" daemon prio=5 tid=7 Blocked
  | group="system" sCount=1 dsCount=0 obj=0x12c31940 self=0x762068467600
  | sysTid=29880 nice=0 cgrp=default sched=0/0 handle=0x76204fa70450
  | state=S schedstat=( 0 0 0 ) utm=2 stm=0 core=0 HZ=100
  | stack=0x76204f96e000-0x76204f970000 stackSize=1037KB
  | held mutexes=
  kernel: futex_wait_queue_me+0xdb/0x123
  kernel: futex_wait+0x11c/0x261
  kernel: do_futex+0xc7/0x999
  kernel: SyS_futex+0xbd/0x124
  kernel: system_call_fastpath+0x16/0x1b
  native: #00 pc 000000000001ccc8  /system/lib64/libc.so (syscall+24)
  native: #01 pc 000000000014f6ae  /system/lib64/libart.so (_ZN3art17ConditionVariable9TimedWaitEPNS_6ThreadEli+142)
  native: #02 pc 00000000002c1dbd  /system/lib64/libart.so (_ZN3art2gc13TaskProcessor7GetTaskEPNS_6ThreadE+269)
  native: #03 pc 00000000002c264f  /system/lib64/libart.so (_ZN3art2gc13TaskProcessor11RunAllTasksEPNS_6ThreadE+63)
  native: #04 pc 00000000001e7ca1  /system/framework/x86_64/boot-core-libart.oat (Java_dalvik_system_VMRuntime_runHeapTasks__+157)
  at dalvik.system.VMRuntime.runHeapTasks(Native method)
  - waiting to lock an unknown object
  at java.lang.Daemons$HeapTaskDaemon.run(Daemons.java:433)
  at java.lang.Thread.run(Thread.java:761)

"FinalizerWatchdogDaemon" daemon prio=5 tid=8 Waiting
  | group="system" sCount=1 dsCount=0 obj=0x12c318b0 self=0x762068466c00
  | sysTid=29879 nice=0 cgrp=default sched=0/0 handle=0x76204fb75450
  | state=S schedstat=( 0 0 0 ) utm=0 stm=0 core=0 HZ=100
  | stack=0x76204fa73000-0x76204fa75000 stackSize=1037KB
  | held mutexes=
  at java.lang.Object.wait!(Native method)
  - waiting on <0x0a509a51> (a java.lang.Daemons$FinalizerWatchdogDaemon)
  at java.lang.Daemons$FinalizerWatchdogDaemon.sleepUntilNeeded(Daemons.java:269)
  - locked <0x0a509a51> (a java.lang.Daemons$FinalizerWatchdogDaemon)
  at java.lang.Daemons$FinalizerWatchdogDaemon.run(Daemons.java:249)
  at java.lang.Thread.run(Thread.java:761)

"main" prio=5 tid=1 Runnable
  | group="main" sCount=0 dsCount=0 obj=0x75461ed0 self=0x762068495a00
  | sysTid=29869 nice=0 cgrp=default sched=0/0 handle=0x76206ce66b40
  | state=R schedstat=( 0 0 0 ) utm=7 stm=3 core=0 HZ=100
  | stack=0x7fff365dc000-0x7fff365de000 stackSize=8MB
  | held mutexes= "mutator lock"(shared held)
  at mlh.<init>(:com.google.android.gms@11947480:2)
  at mlh.<init>(:com.google.android.gms@11947480:1)
  at com.google.android.gms.backup.BackupTransportChimeraService.onCreate(:com.google.android.gms@11947480:16)
  at csc.onCreate(:com.google.android.gms@11947480:1)
  at ldw.onCreate(:com.google.android.gms@11947480:3)
  at android.app.ActivityThread.handleCreateService(ActivityThread.java:3191)
  at android.app.ActivityThread.-wrap5(ActivityThread.java:-1)
  at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1567)
  at android.os.Handler.dispatchMessage(Handler.java:102)
  at android.os.Looper.loop(Looper.java:154)
  at android.app.ActivityThread.main(ActivityThread.java:6119)
  at java.lang.reflect.Method.invoke!(Native method)
  at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:886)
  at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:776)

"FinalizerDaemon" daemon prio=5 tid=9 Waiting
  | group="system" sCount=1 dsCount=0 obj=0x12c31820 self=0x7620684c1e00
  | sysTid=29878 nice=0 cgrp=default sched=0/0 handle=0x76206760a450
  | state=S schedstat=( 0 0 0 ) utm=0 stm=0 core=0 HZ=100
  | stack=0x762067508000-0x76206750a000 stackSize=1037KB
  | held mutexes=
  at java.lang.Object.wait!(Native method)
  - waiting on <0x0efc92b6> (a java.lang.Object)
  at java.lang.Object.wait(Object.java:407)
  at java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:188)
  - locked <0x0efc92b6> (a java.lang.Object)
  at java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:209)
  at java.lang.Daemons$FinalizerDaemon.run(Daemons.java:204)
  at java.lang.Thread.run(Thread.java:761)

"ReferenceQueueDaemon" daemon prio=5 tid=10 Waiting
  | group="system" sCount=1 dsCount=0 obj=0x12c31790 self=0x762068498200
  | sysTid=29877 nice=0 cgrp=default sched=0/0 handle=0x76206770f450
  | state=S schedstat=( 0 0 0 ) utm=0 stm=0 core=0 HZ=100
  | stack=0x76206760d000-0x76206760f000 stackSize=1037KB
  | held mutexes=
  at java.lang.Object.wait!(Native method)
  - waiting on <0x0ce1bf24> (a java.lang.Class<java.lang.ref.ReferenceQueue>)
  at java.lang.Daemons$ReferenceQueueDaemon.run(Daemons.java:150)
  - locked <0x0ce1bf24> (a java.lang.Class<java.lang.ref.ReferenceQueue>)
  at java.lang.Thread.run(Thread.java:761)

"GlobalDispatchingHandlerThread" prio=5 tid=11 Native
  | group="main" sCount=1 dsCount=0 obj=0x12d27dc0 self=0x762068469400
  | sysTid=30042 nice=9 cgrp=default sched=0/0 handle=0x7620450e7450
  | state=S schedstat=( 0 0 0 ) utm=0 stm=0 core=0 HZ=100
  | stack=0x762044fe5000-0x762044fe7000 stackSize=1037KB
  | held mutexes=
  kernel: SyS_epoll_wait+0x358/0x402
  kernel: SyS_epoll_pwait+0x81/0xf3
  kernel: system_call_fastpath+0x16/0x1b
  native: #00 pc 000000000008c6ea  /system/lib64/libc.so (__epoll_pwait+10)
  native: #01 pc 000000000002a165  /system/lib64/libc.so (epoll_pwait+53)
  native: #02 pc 00000000000185e6  /system/lib64/libutils.so (_ZN7android6Looper9pollInnerEi+166)
  native: #03 pc 000000000001847a  /system/lib64/libutils.so (_ZN7android6Looper8pollOnceEiPiS1_PPv+42)
  native: #04 pc 0000000000104455  /system/lib64/libandroid_runtime.so (_ZN7android18NativeMessageQueue8pollOnceEP7_JNIEnvP8_jobjecti+37)
  native: #05 pc 000000000087cd6f  /system/framework/x86_64/boot-framework.oat (Java_android_os_MessageQueue_nativePollOnce__JI+187)
  at android.os.MessageQueue.nativePollOnce(Native method)
  at android.os.MessageQueue.next(MessageQueue.java:323)
  at android.os.Looper.loop(Looper.java:136)
  at android.os.HandlerThread.run(HandlerThread.java:61)

"FileObserver" prio=5 tid=12 Native
  | group="main" sCount=1 dsCount=0 obj=0x12d39310 self=0x76205f1ba600
  | sysTid=30052 nice=0 cgrp=default sched=0/0 handle=0x762044fe2450
  | state=S schedstat=( 0 0 0 ) utm=0 stm=0 core=0 HZ=100
  | stack=0x762044ee0000-0x762044ee2000 stackSize=1037KB
  | held mutexes=
  kernel: inotify_read+0x255/0x2a5
  kernel: vfs_read+0xa9/0x132
  kernel: SyS_read+0x4b/0x73
  kernel: system_call_fastpath+0x16/0x1b
  native: #00 pc 000000000008d585  /system/lib64/libc.so (read+5)
  native: #01 pc 0000000000162e30  /system/lib64/libandroid_runtime.so (???)
  native: #02 pc 000000000084d53f  /system/framework/x86_64/boot-framework.oat (Java_android_os_FileObserver_00024ObserverThread_observe__I+171)
  at android.os.FileObserver$ObserverThread.observe(Native method)
  at android.os.FileObserver$ObserverThread.run(FileObserver.java:86)

"GoogleApiHandler" prio=5 tid=13 Native
  | group="main" sCount=1 dsCount=0 obj=0x12d51310 self=0x762068468000
  | sysTid=30077 nice=9 cgrp=default sched=0/0 handle=0x762044edd450
  | state=S schedstat=( 0 0 0 ) utm=1 stm=0 core=0 HZ=100
  | stack=0x762044ddb000-0x762044ddd000 stackSize=1037KB
  | held mutexes=
  kernel: SyS_epoll_wait+0x358/0x402
  kernel: SyS_epoll_pwait+0x81/0xf3
  kernel: system_call_fastpath+0x16/0x1b
  native: #00 pc 000000000008c6ea  /system/lib64/libc.so (__epoll_pwait+10)
  native: #01 pc 000000000002a165  /system/lib64/libc.so (epoll_pwait+53)
  native: #02 pc 00000000000185e6  /system/lib64/libutils.so (_ZN7android6Looper9pollInnerEi+166)
  native: #03 pc 000000000001847a  /system/lib64/libutils.so (_ZN7android6Looper8pollOnceEiPiS1_PPv+42)
  native: #04 pc 0000000000104455  /system/lib64/libandroid_runtime.so (_ZN7android18NativeMessageQueue8pollOnceEP7_JNIEnvP8_jobjecti+37)
  native: #05 pc 000000000087cd6f  /system/framework/x86_64/boot-framework.oat (Java_android_os_MessageQueue_nativePollOnce__JI+187)
  at android.os.MessageQueue.nativePollOnce(Native method)
  at android.os.MessageQueue.next(MessageQueue.java:323)
  at android.os.Looper.loop(Looper.java:136)
  at android.os.HandlerThread.run(HandlerThread.java:61)

"SharedPreferencesImpl-load" prio=5 tid=14 Native
  | group="main" sCount=1 dsCount=0 obj=0x12cb25e0 self=0x76205d24a400
  | sysTid=30131 nice=0 cgrp=default sched=0/0 handle=0x762044dd8450
  | state=S schedstat=( 0 0 0 ) utm=0 stm=0 core=0 HZ=100
  | stack=0x762044cd6000-0x762044cd8000 stackSize=1037KB
  | held mutexes=
  kernel: futex_wait_queue_me+0xdb/0x123
  kernel: futex_wait+0x11c/0x261
  kernel: do_futex+0xc7/0x999
  kernel: SyS_futex+0xbd/0x124
  kernel: system_call_fastpath+0x16/0x1b
  native: #00 pc 000000000001ccc6  /system/lib64/libc.so (syscall+22)
  native: #01 pc 000000000014f12b  /system/lib64/libart.so (_ZN3art17ConditionVariable16WaitHoldingLocksEPNS_6ThreadE+123)
  native: #02 pc 0000000000172546  /system/lib64/libart.so (_ZN3art8CheckJNI29ReleasePrimitiveArrayElementsEPKcNS_9Primitive4TypeEP7_JNIEnvP7_jarrayPvi+454)
  native: #03 pc 000000000002a0d2  /system/lib64/libjavacore.so (???)
  native: #04 pc 00000000002c37b7  /system/framework/x86_64/boot-core-libart.oat (Java_libcore_io_Posix_readBytes__Ljava_io_FileDescriptor_2Ljava_lang_Object_2II+259)
  at libcore.io.Posix.readBytes(Native method)
  at libcore.io.Posix.read(Posix.java:169)
  at libcore.io.BlockGuardOs.read(BlockGuardOs.java:231)
  at libcore.io.IoBridge.read(IoBridge.java:471)
  at java.io.FileInputStream.read(FileInputStream.java:252)
  at java.io.BufferedInputStream.fill(BufferedInputStream.java:235)
  at java.io.BufferedInputStream.read1(BufferedInputStream.java:275)
  at java.io.BufferedInputStream.read(BufferedInputStream.java:334)
  - locked <0x0e338a8d> (a java.io.BufferedInputStream)
  at sun.nio.cs.StreamDecoder.readBytes(StreamDecoder.java:287)
  at sun.nio.cs.StreamDecoder.implRead(StreamDecoder.java:350)
  at sun.nio.cs.StreamDecoder.read(StreamDecoder.java:179)
  - locked <0x06372442> (a java.io.InputStreamReader)
  at java.io.InputStreamReader.read(InputStreamReader.java:184)
  at org.kxml2.io.KXmlParser.fillBuffer(KXmlParser.java:1516)
  at org.kxml2.io.KXmlParser.peekCharacter(KXmlParser.java:1470)
  at org.kxml2.io.KXmlParser.setInput(KXmlParser.java:1746)
  at com.android.internal.util.XmlUtils.readMapXml(XmlUtils.java:739)
  at android.app.SharedPreferencesImpl.loadFromDisk(SharedPreferencesImpl.java:130)
  at android.app.SharedPreferencesImpl.-wrap1(SharedPreferencesImpl.java:-1)
  at android.app.SharedPreferencesImpl$1.run(SharedPreferencesImpl.java:100)

"SharedPreferencesImpl-load" prio=5 tid=15 Native
  | group="main" sCount=1 dsCount=0 obj=0x12cbd3a0 self=0x76205d24ae00
  | sysTid=30132 nice=0 cgrp=default sched=0/0 handle=0x762044cd3450
  | state=S schedstat=( 0 0 0 ) utm=0 stm=0 core=0 HZ=100
  | stack=0x762044bd1000-0x762044bd3000 stackSize=1037KB
  | held mutexes=
  kernel: futex_wait_queue_me+0xdb/0x123
  kernel: futex_wait+0x11c/0x261
  kernel: do_futex+0xc7/0x999
  kernel: SyS_futex+0xbd/0x124
  kernel: system_call_fastpath+0x16/0x1b
  native: #00 pc 000000000001ccc6  /system/lib64/libc.so (syscall+22)
  native: #01 pc 000000000014f12b  /system/lib64/libart.so (_ZN3art17ConditionVariable16WaitHoldingLocksEPNS_6ThreadE+123)
  native: #02 pc 0000000000172546  /system/lib64/libart.so (_ZN3art8CheckJNI29ReleasePrimitiveArrayElementsEPKcNS_9Primitive4TypeEP7_JNIEnvP7_jarrayPvi+454)
  native: #03 pc 000000000002a0d2  /system/lib64/libjavacore.so (???)
  native: #04 pc 00000000002c37b7  /system/framework/x86_64/boot-core-libart.oat (Java_libcore_io_Posix_readBytes__Ljava_io_FileDescriptor_2Ljava_lang_Object_2II+259)
  at libcore.io.Posix.readBytes(Native method)
  at libcore.io.Posix.read(Posix.java:169)
  at libcore.io.BlockGuardOs.read(BlockGuardOs.java:231)
  at libcore.io.IoBridge.read(IoBridge.java:471)
  at java.io.FileInputStream.read(FileInputStream.java:252)
  at java.io.BufferedInputStream.fill(BufferedInputStream.java:235)
  at java.io.BufferedInputStream.read1(BufferedInputStream.java:275)
  at java.io.BufferedInputStream.read(BufferedInputStream.java:334)
  - locked <0x0f92bc53> (a java.io.BufferedInputStream)
  at sun.nio.cs.StreamDecoder.readBytes(StreamDecoder.java:287)
  at sun.nio.cs.StreamDecoder.implRead(StreamDecoder.java:350)
  at sun.nio.cs.StreamDecoder.read(StreamDecoder.java:179)
  - locked <0x020da990> (a java.io.InputStreamReader)
  at java.io.InputStreamReader.read(InputStreamReader.java:184)
  at org.kxml2.io.KXmlParser.fillBuffer(KXmlParser.java:1516)
  at org.kxml2.io.KXmlParser.peekCharacter(KXmlParser.java:1470)
  at org.kxml2.io.KXmlParser.setInput(KXmlParser.java:1746)
  at com.android.internal.util.XmlUtils.readMapXml(XmlUtils.java:739)
  at android.app.SharedPreferencesImpl.loadFromDisk(SharedPreferencesImpl.java:130)
  at android.app.SharedPreferencesImpl.-wrap1(SharedPreferencesImpl.java:-1)
  at android.app.SharedPreferencesImpl$1.run(SharedPreferencesImpl.java:100)

----- end 29869 -----
```

**Comment by chickenbeef** [comment]
Noticing the same here, using the butomo1989/docker-android-x86-7.1.1 image on Ubuntu 16.04.

I cannot launch some of our test apps as well, the emulator just restarts when attempting to launch the app.

Switched back to the butomo1989/docker-android-x86-7.0 image and it's stable.

**Comment by Kignuf** [comment]
Thanks @chickenbeef it does work on 7.0 emulators

**Comment by red-avtovo** [comment]
The other solution - is to open `Settings` from Notification center in top right corner by tapping gear button

**Comment by chickenbeef** [comment]
@red-avtovo I think you mean workaround :)

**Comment by red-avtovo** [comment]
This issue is related to `x86_64` images, which are not that stable as it should be. 
Here is an explanation of the issue:
https://github.com/butomo1989/docker-android/issues/82#issuecomment-402542754

Try to wait for the next release and use fresh `x86` images, which would have `x86` images inside of it

**Comment by butomo1989** [comment]
Hi @Kignuf @chickenbeef @red-avtovo , 

could you guys try it again with the new image? I just released the new version yesterday and checked that there is no issue with settings anymore.

**Comment by red-avtovo** [comment]
@butomo1989 I'm checking [7.1.1](https://hub.docker.com/r/butomo1989/docker-android-x86-7.1.1/) right now. Didn't find Android 8.0, 8.1 though

**Comment by red-avtovo** [comment]
7.1.1 works like a charm

**Comment by red-avtovo** [comment]
Checked 8.0 works as well

There is only 2 problems, which I encountered:
1) Appium ignores `port` specification and listens to 4723 all the time (Most likely Appium issue)
2) On Android 8 we should accept USB debugging right after we created the container. Hope, there is a way, how to automate that

<img width="425" alt="image" src="https://user-images.githubusercontent.com/10682826/42372196-e417aa96-8111-11e8-870b-7b614d87bef2.png">

**Comment by butomo1989** [comment]
Thank you @red-avtovo. hmm.. Regarding USB debugging I think there is issue with Emulator. we can create a separate ticket. I will close this ticket then because I want to clean the open issues a little bit. :b
